package.json

```diff
@@ -27,12 +26,11 @@
   },
   "homepage": "https://github.com/lgeiger/node-abi#readme",
   "devDependencies": {
-    "got": "^10.6.0",
-    "semantic-release": "^15.8.0",
-    "tape": "^4.6.3",
-    "travis-deploy-once": "^5.0.1"
+    "@continuous-auth/semantic-release-npm": "^2.0.0",
+    "got": "^11.8.2",
+    "tape": "^5.3.1"
   },
   "dependencies": {
-    "semver": "^5.4.1"
+    "semver": "^7.3.5"
```

Comment:
This is a breaking change, semver@7.0.0 requires Node 10. To be fair I have no idea what the minimum version of Node required for this module is :sweat_smile: 

We should take the change, but we should also add `engines` appropriately.

Reply:
Oof, good catch, thanks! I'll add the engines to check for 10 or higher - that's required for our CircleCI jobs anyway, and anything under Node 10 would be fairly old.

Reply:
I'm also fine with us artificially requiring Node 12 since Node 10 is EOL'd.